components/resource_control/src/resource_group.rs

```diff
@@ -464,7 +464,7 @@ impl ResourceController {
 
     pub fn update_min_virtual_time(&self) {
         let start = Instant::now_coarse();
-        let mut min_vt = u64::MAX;
+        let mut min_vt = u64::MAX - 100_000;
```

Comment:
why

Reply:
Oh, just for test. Seems like the compiler cannot recognize there logic and reports an error `attempt to add with overflow` at `min_vt + 100_000`.
I will change it to `min_vt.wrapping_add(100_000)` to make it happy.